index.html

```diff
@@ -78,11 +78,11 @@ <h1 class="project-title">Tonic</h1>
             </div>
 
             <div class="projects project2">
-                <img class ='project-image' src="images/project2.png" alt="">
+                <img class ='project-image' src="images/project2.png" alt="screenshot of project two">
 
                 <div class="projects-describe">
                     <h1 class="project-title">Multi-Post<br>Stories</h1>
```

Comment:
- [ ]  In order to make your project more semantically correct and accessible, it would be a good thing to limit your page to only one ```<h1>``` tag per page. This said, kindly change all the other ```<h1>``` elements as well and only remain with the main header which is at the top of the website at the landing page.